<launch>

	<node pkg="vicon" type="vicon_node" name="vicon" respawn="true">
		<param name="vicon_IP" value="192.168.1.4:801" />
	</node>

	<group ns="gandhi">
	<node pkg="to_velocity_tracking" type="control_node" name="control">
		<rosparam param="robot_list">["gandhi", "suleyman", "alexander", "hiawatha"]</rosparam>
		<param name="robotname" value="gandhi" />
		<param name="k" type="double" value="0.2" />
 		<param name="ktheta" type="double" value="1.0" />
 		<param name="power" type="boolean" value="false" />
 	</node>
 	
	<node pkg="create_driver" type="create_driver" name="driver" respawn="true">
		<param name="hostname" value="192.168.1.20" />
	</node>
	</group>
	
	<group ns="suleyman">
	<node pkg="to_velocity_tracking" type="control_node" name="control">
		<rosparam param="robot_list">["gandhi", "suleyman", "alexander", "hiawatha"]</rosparam>
		<param name="robotname" value="suleyman" />
		<param name="k" type="double" value="0.2" />
 		<param name="ktheta" type="double" value="1.0" />
 		<param name="power" type="boolean" value="false" />
 	</node>
 	
 	<node pkg="create_driver" type="create_driver" name="driver" respawn="true">
		<param name="hostname" value="192.168.1.26" />
	</node>
 	</group>
 	
 	<group ns="alexander"> 	
	<node pkg="to_velocity_tracking" type="control_node" name="control">
		<rosparam param="robot_list">["gandhi", "suleyman", "alexander", "hiawatha"]</rosparam>
		<param name="robotname" value="alexander" />
		<param name="k" type="double" value="0.2" />
 		<param name="ktheta" type="double" value="1.0" />
 		<param name="power" type="boolean" value="false" />
 	</node>
 	
 	<node pkg="create_driver" type="create_driver" name="driver" respawn="true">
		<param name="hostname" value="192.168.1.28" />
	</node>	
 	</group>
 	
 	<group ns="hiawatha">
	<node pkg="to_velocity_tracking" type="control_node" name="control">
		<rosparam param="robot_list">["gandhi", "suleyman", "alexander", "hiawatha"]</rosparam>
		<param name="robotname" value="hiawatha" />
		<param name="k" type="double" value="0.2" />
 		<param name="ktheta" type="double" value="1.0" />
 		<param name="power" type="boolean" value="false" />
 	</node> 
 	
 	<node pkg="create_driver" type="create_driver" name="driver" respawn="true">
		<param name="hostname" value="192.168.1.24" />
	</node>	
 	</group>	 
 	
 	<node pkg="rqt_reconfigure" type="rqt_reconfigure" name="reconfigure_gui"/>
	
	
</launch>
